cart.html

```diff
@@ -0,0 +1,581 @@
+                          fill="#05B999" />
+                      </g>
+                      <defs>
+                        <clipPath id="clip0_2923_2129">
+                          <rect width="16" height="16" fill="white" transform="translate(0 0.418335)" />
+                        </clipPath>
+                      </defs>
+                    </svg> Disponible para envío gratis
+                  </p>
+                </div>
+              </div>
+              <button class="cart__information-purchase-summary-price-btn-buy">PROCESAR EL PAGO <svg
+                  xmlns="http://www.w3.org/2000/svg" width="17" height="17" viewBox="0 0 17 17" fill="none">
+                  <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M2.91589 9.34584C2.91589 8.24127 3.81132 7.34583 4.91589 7.34583H12.9159C14.0205 7.34583 14.9159 8.24127 14.9159 9.34583V12.6792C14.9159 13.7837 14.0205 14.6792 12.9159 14.6792H4.91589C3.81132 14.6792 2.91589 13.7837 2.91589 12.6792V9.34584Z"
+                    stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                  <path
+                    d="M5.58252 7.34583V4.67917C5.58252 2.83822 7.0749 1.34583 8.91585 1.34583C10.7568 1.34583 12.2492 2.83822 12.2492 4.67917V7.34583"
+                    stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="cart__products-additionals">
+        <div class="container">
+          <p class="cart__products-additionals-title">Te podría interesar</p>
```

Comment:
Esto tambien deberia ser un heading
```suggestion
          <h3 class="cart__products-additionals-title">Te podría interesar</h3>
```